Source/ASNetworkImageNode.mm

```diff
@@ -107,7 +109,7 @@ - (instancetype)init
 
 - (void)dealloc
 {
-  [self _cancelImageDownload];
+  [self _cancelImageDownload:NO];
```

Comment:
The new boolean param passed to `_cancelImageDownload:` is not descriptive and thus a bit confusing.  How about `_cancelImageDownloadWithResumePossibility:`?  Same concern for `_cancelDownloadAndClearImage:` and their `_locked` variants.